pkgs/by-name/sm/smtp4dev/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  buildDotnetModule,
+  fetchFromGitHub,
+  nodejs,
+  npmHooks,
+  fetchNpmDeps,
+  dotnetCorePackages,
+}:
+let
+  version = "3.3.4";
+  src = fetchFromGitHub {
+    owner = "rnwood";
+    repo = "smtp4dev";
+    rev = "ref/tags/${version}";
```

```suggestion
    rev = "refs/tags/${version}";
```

Without this I get a 404 when downloading the source:
```
error: builder for '/nix/store/8sfq45bnid91nk1i546h7m7gzhlnj937-source.drv' failed with exit code 1;
       last 8 log lines:
       >
       > trying https://github.com/rnwood/smtp4dev/archive/ref/tags/3.3.4.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix log /nix/store/8sfq45bnid91nk1i546h7m7gzhlnj937-source.drv'.
error: 1 dependencies of derivation '/nix/store/6znyavj7gzrix98xa5s3qpjans2ymva4-smtp4dev-3.3.4.drv' failed to build
```